tools/st2ctl

```diff
@@ -122,6 +122,7 @@ function st2start(){
   else
     service mistral start
   fi
+  (cd /opt/stackstorm/static/webui && nohup python -m SimpleHTTPServer 80) &>> ${LOGFILE} &
```

Comment:
This is fine for now.  Honestly, we just need to deprecate this damn script and move to upstart/init/etc